### DISCUSSION TITLE
Easy to implement suggestion

### DISCUSSION THREAD

**Discussion by brunoCreator** [Yuu6883/MarkovJuniorWeb#6] [Ideas]
This could be easily an editor

# easy to add features
- edit the XML file
- change output height and width

* Some simple run options buttons that can be added
-New run- clean, randomize new seed and run 
-Finish- run at max speed util finished
-download the edited XML file

* also add the technical notes in a tab, https://gist.github.com/dogles/a926ab890552cc7e45400a930398449d

# More complex features
- simple pixel art editor to pre edit the image and to edit tiles or visualizer in and out
- Edit directly the visualizer proprieties.
- toolbar to add new steps or remove.

# Can go even further
- could use a simple free PHP+MySQL server to save and share online, with a simple register and login, and a basic CRUD

**Comment by Yuu6883** [comment]
Those are good ideas, but I might not have the capacity for personal projects anymore - anyone should feel free to contribute to this project 👍
